build-config/resources/BallerinaTest.toml

```diff
@@ -4,4 +4,9 @@ name = "http_tests"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
+scope = "testOnly"
```

Comment:
why is `http-native` test only?

Reply:
Ah this is ballerina test module.

Reply:
This is for the ballerina http test module which only have ballerina tests